Makefile

```diff
@@ -3,13 +3,15 @@ TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
 include ./Makefile.os
 
 GITHUB_VERSION ?= main
+OAUTH_TAG = main
 RELEASE_VERSION ?= latest
 CHART_SEMANTIC_RELEASE_VERSION ?= $(shell cat ./release.version | tr A-Z a-z)
 BRIDGE_VERSION ?= $(shell cat ./bridge.version | tr A-Z a-z)
 DOCKER_CMD ?= docker
 
 ifneq ($(RELEASE_VERSION),latest)
   GITHUB_VERSION = $(RELEASE_VERSION)
+  OAUTH_TAG = $(shell source ./tools/strimzi-oauth-version.sh && get_strimzi_oauth_version)
```

Comment:
Strimzi operators main branch does not use main branch of OAuth. It is always linked to a specific OAuth release. So this should be set this way by default and not just for a release.